jobs/build/rhcos_sync/Jenkinsfile

```diff
@@ -119,19 +120,34 @@ node {
         }
 
         tag = params.FROM_RELEASE_TAG
-        if (params.OVERRIDE_NAME != "") {
-            name = params.OVERRIDE_NAME
-        } else {
-            name = tag
```

Comment:
Specifying a tag, which seemed like it ought to be how this is normally done, was full of pitfalls. Here, the default name is the same as the tag which would include the arch suffix. Rather than try to sort that out, take the default from the name in the release image itself.